### PR TITLE
docs: docker compose v2 notes

### PR DESCRIPTION
**Pull request by strophy**
## Issue being fixed or feature implemented

Dashmate does not work properly with Docker Compose v2. 

## What was done?

Write a note in the readme on how to ensure you are using Docker Compose v1. Also bumped node minimum versions.

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,10 +32,12 @@ Distribution package for Dash Masternode installation
 
 * [Docker](https://docs.docker.com/engine/installation/) (v18.06.0+)
 * [Docker Compose](https://docs.docker.com/compose/install/) (v1.25.0+)
-* [Node.js](https://nodejs.org/en/download/) (v10.0+)
+* [Node.js](https://nodejs.org/en/download/) (v16.0+, NPM v8.0+)
```
Comment (shumkov) [inline README.md:35]: Is it our requirements? I guess it still work on 12.

File: README.md [hunk 2]
```diff
@@ -32,10 +32,12 @@ Distribution package for Dash Masternode installation
...
 
 For Linux installations you may optionally wish to follow the [post-installation steps](https://docs.docker.com/engine/install/linux-postinstall/) to manage Docker as a non-root user, otherwise you will have to run CLI and Docker commands with `sudo`.
 
+Note that Docker Compose v2 is not yet supported. Docker Desktop will automatically enable Docker Compose v2 under Windows and macOS, please go to "Settings -> General -> Use Docker Compose V2" and disable the checkbox to use Dashmate.
```
Comment (shumkov) [inline README.md]: Let’s make it bold or highlight it somehow?

File: README.md (hunk 1 above)
Comment (jawid-h) [reply README.md:35]: @shumkov The lock file is of version 2 now, I don't think it'll work properly with older NodeJS versions.

**Review by jawid-h — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (strophy) [reply README.md:35]: NPM 8 still supports Node 12, so I guess we could change it like that

**Pull request merged**